stdlib/public/core/Duration.swift

```diff
@@ -108,9 +108,13 @@ extension Duration {
   ///
   /// - Returns: A `Duration` representing a given number of seconds.
   @available(SwiftStdlib 5.7, *)
+  @inlinable
   public static func seconds<T: BinaryInteger>(_ seconds: T) -> Duration {
-    return Duration(_attoseconds:
-      _Int128(seconds).multiplied(by: 1_000_000_000_000_000_000 as UInt64))
+    guard let high = Int64(exactly: seconds >> 64) else { fatalError() }
```

Comment:
`if #available(...) { use Int128 }`?

Reply:
Not possible yet, because of how availability checking works (or doesn't) for calls from a library to itself and the way 9999 is implemented. We can do that at some future point, however.